server/bundler/loader.js

```diff
@@ -33,6 +35,11 @@ function getSectionsModule( sections ) {
 		'	},',
 		'	load: function() {',
 		'		' + sectionLoaders,
+		'	},',
+		'	preload: function( section ) {',
+		'		switch ( section ) {',
```

Comment:
Discussed in chat more, but the gist of it is that we need to construct the `require.ensure` at build time so webpack can properly transform them.